configure.in

```diff
@@ -313,6 +315,7 @@ AC_CHECK_LIB(rt, sem_init, , [
 ])
 AC_CHECK_LIB(posix4, sem_wait, , )
 AC_CHECK_LIB(pthread, pthread_create, , )
+AC_CHECK_LIB(flatccrt, flatcc_builder_init, , )
```

Comment:
I think we should force them to be there.  If they aren't we're forever responsible for the system working both with and without them which seems unnecessarily burdensome.